src/python/ImpactX.cpp

```diff
@@ -103,8 +105,17 @@ void init_ImpactX (py::module& m)
         )
 
         // from amrex::AmrMesh
-        .def_property_readonly("max_level",
+        .def_property("max_level",
             [](ImpactX & ix){ return ix.maxLevel(); },
+            [](ImpactX & /* ix */, int max_level) {
+                throw std::runtime_error("setting n_cell is not yet implemented");
```

Comment:
Not yet working, needs a bit of a quirk in `ImpactX.cpp` as in #402 that I have not yet figured out :)